src/utils/detect-server-settings.mjs

```diff
@@ -259,9 +261,6 @@ const detectFrameworkSettings = async ({ projectDir }) => {
   }
 
   if (frameworks.length > 1) {
-    // performance optimization, load inquirer on demand
```

Comment:
if this was here as a performance optimisation, maybe it should stay as a `dynamic import`. @lukasholzer what do you think?

Reply:
TBH I would benchmark it and test it locally if it makes a difference. If not than we can move forward with a static one otherwise I would leave it a dynamic one.